services/docker-stack-bo-ps.yml

```diff
@@ -211,34 +141,24 @@ services:
       - ioc-network
 
   corrs-ia07:
-      image: dockerregistry.lnls-sirius.com.br/fac/fac-iocs:__FAC_IOC_TAG_TEMPLATE__
-      depends_on:
-        - "dips"
-      command: bash -c 'source /etc/profile && /ioc-apps/bo-ps-corrs-ia07.bash'
-      deploy:
-        placement:
-          constraints:
-            # - node.hostname != CA-RaTim-CO-IOCSrv
-            # - node.hostname != IA-14RaDiag03-CO-IOCSrv
-            # - node.hostname != IA-16RaBbB-CO-IOCSrv
-            # - node.hostname != IA-18RaDiag04-CO-IOCSrv
-            # - node.hostname != IA-20RaDiag01-CO-IOCSrv-1
-            - node.hostname != IA-20RaDiag01-CO-IOCSrv-2
-            # - node.hostname != IA-20RaDiag02-CO-IOCSrv-1
-            # - node.hostname != IA-20RaDiag02-CO-IOCSrv-2
-            - node.hostname != CA-RaCtrl-CO-Srv-1
-            - node.hostname != LA-RaCtrl-CO-Srv-1
-            - node.hostname != TA-TiRack-CO-FWSrv-1
-        replicas: 1
-        restart_policy:
-          condition: any
-      logging:
-        driver: "json-file"
-        options:
-          max-file: "10"
-          max-size: "10m"
-      networks:
-        - ioc-network
+    image: dockerregistry.lnls-sirius.com.br/fac/fac-iocs:__FAC_IOC_TAG_TEMPLATE__
+    depends_on:
+      - "dips"
+    command: bash -c 'source /etc/profile && /ioc-apps/bo-ps-corrs-ia07.bash'
+    deploy:
+      placement:
+        constraints:
+          - node.hostname == lnlsfac-srv2
+      replicas: 1
+      restart_policy:
+        condition: any
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "10"
+        max-size: "10m"
+    networks:
+      - ioc-network
```

Comment:
idem here

Reply:
yes, before committing I noticed these two "07" file diffs. I could not find why git diff differentiated these two. My guess is that original versions of these files (edited manually) were different from the rest. But I did not put time to pinpoint it, since I checked other files (which work) - diff-comparing with original files - and they all checked Ok.